core/src/main/java/org/elasticsearch/search/aggregations/metrics/tophits/TopHitsBuilder.java

```diff
@@ -366,7 +186,12 @@ private SearchSourceBuilder sourceBuilder() {
         return sourceBuilder;
     }
 
-    public HighlightBuilder highlightBuilder() {
+    public BytesReference highlighter() {
```

Comment:
But when we have refactored the HighlighterBuilders we will want this getter and I am worried that if we remove this method here we will forget to added the getter after refactoring